x/airdrop/simulation/operations.go

```diff
@@ -20,6 +20,8 @@ import (
 const (
 	OpWeightMsgCreateAirdropAccount = "op_weight_msg_create_airdrop_account" //nolint: gosec
 	OpWeightMsgClaimAirdrop         = "op_weight_msg_claim_airdrop"          //nolint: gosec
+
+	DefaultWeightMsgSend = 100 // from simappparams.DefaultWeightMsgSend
```

Comment:
Why is the default weight for airdrop sims 100?

Iirc it was 100 for the oracle module since prevotes & votes have to go in always for any expected behavior

Reply:
This value used to be taken from simapp/params package (https://github.com/cosmos/cosmos-sdk/blob/2a48f6d7a24b728ac64eafe0929bb8893cc23ccf/simapp/params/weights.go#L5) but that got deprecated. Its set to be to the same value it was before